.github/CODEOWNERS

```diff
@@ -115,6 +115,9 @@
 /pkg/sql/zone*.go            @cockroachdb/sql-foundations
 /pkg/cmd/sql-bootstrap-data/ @cockroachdb/sql-foundations
 
+#!/pkg/sql/logictest/tests/cockroach-go-testserver-*/*.go @cockroachdb/sql-foundations-noreview
```

Comment:
what is the `#!` syntax? i didn't see it documented from a quick glance at https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

Reply:
IIUC it's our custom rule for packages that have owners but changes to which shouldn't result in PR requests https://github.com/cockroachdb/cockroach/blob/5b1a3c18174302d23dd3c2f3616fa86c7ac00396/.github/CODEOWNERS#L8-L11

Reply:
nice!

i think the team name here should still be `sql-foundations` and not `sql-foundations-noreview`. the latter doesn't exist, unless there's also another special rule for the `-noreview` suffix?

Reply:
`-noreview` prefix is trimmed here https://github.com/cockroachdb/cockroach/blob/3d2785d8b97a5fc972b26fff574534beb0012622/pkg/internal/codeowners/codeowners.go#L69

It seems like it's the canonical way of assigning ownership to a team without requesting the review on PRs. `#!` in the line prefix is required and `-noreview` in the team name makes it more visible. There is a TODO to lint this way https://github.com/cockroachdb/cockroach/blob/5b1a3c18174302d23dd3c2f3616fa86c7ac00396/.github/CODEOWNERS#L16-L17

Reply:
oh great! well if you're up for it, it would be nice to document that stripping behavior at the top of this file, similar to the other comment you pointed me at.